src/components/countWithTrendWidget/countWithTrendWidget.js

```diff
@@ -1,171 +1,62 @@
-    margin-left: 4px;
-  }
-`;
-
-const StyledTrendContainer = styled.div`
-
-  padding: 1em 2px;
-
-  .trend-value,
-  .trend-date {
-    display: block;
-  }
-
-  .trend-value {
-    .fa {
-      color: #0075cd;
-    }
-  }
-  .trend-date {
-  }
-`;
-
-
-/**
- * Sparkline-esque chart for displaying a Count of latest data and
- * the trend line for that data.
- * @returns {jsx}
- * @constructor
- */
-
-const CountWithTrendWidget = (props) => {
-  console.log(props)
-
-  const {
-    widget: {title, infoText, units},
-    value,
-    trendValue,
-    trendDate,
-  } = props;
-
-
-  const CountValue = ({value, units}) => {
-    if (!value) {
-      return <span className="count-value no-data">No data</span>
-    }
-    return (
-      <span className="count-value">
+  return (
+    <span className="count-value">
         {!units && value}
-        {units === '$' ?
-          <span>{units && <span className="count-units">{units}</span>}{value}</span> :
-          <span>{value}{units && <span className="count-units">{units}</span>}</span>
-        }
+      {units === '$' ?
+        <span>{units && <span className="count-units">{units}</span>}{value}</span> :
+        <span>{value}{units && <span className="count-units">{units}</span>}</span>
+      }
       </span>
-    )
-  };
-
-  const TrendValue = ({value}) => {
-    if (!value) {
-      return null;
-    }
+  )
+};
 
-    console.log(value)
-    return (
-      <span className="trend-value">
+export const TrendValue = ({value}) => {
+  if (!value) {
+    return null;
+  }
+  return (
+    <span className="trend-value">
         {Number(value) > 0 ?
           <span>+{value} <i className="fa fa-arrow-up" /></span> :
           Number(value) < 0 ?
             <span>{value} <i className="fa fa-arrow-down" /></span> :
             <span>-&nbsp;</span>}
       </span>
-    )
-  };
-
-  return (
-    <StyledCount className={`chart--count`} role="article">
-      <StyledHeader className={classnames({
-        'yellow': title.toLowerCase() === 'user satisfaction',
-        'green': title.toLowerCase() === 'cost per transaction',
-        'blue': title.toLowerCase() === 'digital take-up',
-        'purple': title.toLowerCase() === 'completion rate',
-      })}>
-        <h1>{title} {infoText && <Tooltip text={infoText}>?</Tooltip>}</h1>
-      </StyledHeader>
-      <section>
-        <StyledCountContainer>
-          <CountValue value={value} units={units} />
-        </StyledCountContainer>
-        {value && <StyledTrendContainer>
-          <TrendValue value={trendValue} />
-          <span className="trend-date">{trendDate}</span>
-        </StyledTrendContainer>}
-      </section>
-    </StyledCount>
   )
 };
 
+
+/**
+ * A sparkline-esque chart for displaying a Count of latest data and
+ * the trend line for that data.
+ *
+ */
+const CountWithTrendWidget = (props) => {
+  const {viewport, ...rest} = props;
+
+  if (typeof viewport === 'undefined' || viewport === 'sm' || viewport === 'md') {
```

Comment:
So who is responsible for handling browser resize ... datavizkit or the app e.g. dashboard ?